aiida_common_workflows/workflows/relax/gpaw/generator.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+"""Implementation of `aiida_common_workflows.common.relax.generator.CommonRelaxInputGenerator` for GPAW."""
+from typing import Any, Dict, List, Tuple, Union
+import pathlib
+import yaml
+from aiida import engine
+from aiida import orm
+from aiida import plugins
+
+from aiida_common_workflows.common import ElectronicType, RelaxType, SpinType
+from ..generator import CommonRelaxInputGenerator
+
+__all__ = ('GPAWCommonRelaxInputGenerator',)
+
+StructureData = plugins.DataFactory('structure')
```

Comment:
```suggestion
StructureData = plugins.DataFactory('core.structure')
```

Reply:
This is actually my bad. This is the entry point for v2, but forgot that we are still on v1.6 for this project. I will add a commit to remove the `core.` prefix again. Sorry